examples/.gitignore

```diff
@@ -0,0 +1,3 @@
+.gradle
+local.properties
+*.iml
```

Comment:
"build" is already in topmost .gitignore
d2858b561 (Jake Wharton            2012-07-31 00:39:36 -0700 12) build
This decision is out of my scope